### ISSUE TITLE
install.sh creates ~/.config/sourcetrail owned by root

### ISSUE THREAD

**Issue by knatten** [CoatiSoftware/SourcetrailBugTracker#625]
- Downloaded 2018.3.55 for 64 bit Linux from https://www.sourcetrail.com/downloads
- Extracted the archive
- `sudo ./install.sh`
- `sourcetrail` fails:

```
$ sourcetrail
/usr/bin/sourcetrail: 8: [: /bin: unexpected operator
terminate called after throwing an instance of 'boost::filesystem::filesystem_error'
  what():  boost::filesystem::create_directory: Permission denied: "/home/knatten/.config/sourcetrail/log"
Aborted (core dumped)
```

This is because `~/.config/sourcetrail` has wrong ownership:
```
~/.config$ ls -lad sourcetrail/
drwxr-xr-x 2 root root 4096 Oct 11 12:21 sourcetrail/
```

`chown`ing that directory to my user fixed the problem.

**Comment by knatten** [comment]
I don't think that will help, as `$USER` will be `root` when the script is run with sudo. You probably want something like this:
```sh
if [ "$SUDO_USER" ]; then CHOWNUSER=$SUDO_USER; else CHOWNUSER=$USER; fi
```
But I'm not a bash expert, so take this with a grain of salt.

**Comment by egraether** [comment]
Fixed with 2018.3.123
I could reproduce the issue on Ubuntu and your solution worked. Thanks for the help!
We will release a beta build this week.
